src/cache_handler.rs

```diff
@@ -58,7 +54,7 @@ impl Handler for StaticWithCache {
         match requested_path.get_file() {
             Some(file) => {
                 let last_modified_time = match file.stat() {
-                    Err(error) => return Err(box FileError(error) as IronError),
+                    Err(error) => return Err(Box::new(error)),
```

Comment:
@reem, I'm still a bit unsure about the error handling here. Are users supposed to know all possible error types to properly downcast and handle them? And should I create a custom error for this case?